docker-compose.yml

```diff
@@ -13,29 +15,51 @@ services:
       - app
     volumes_from:
       - app
-    environment:
-      - VIRTUAL_HOST
+    env_file:
+      - frontend.env
```

Comment:
this one, I use `.env` which is the default in docker-compose:
https://docs.docker.com/compose/environment-variables/#setting-environment-variables-with-docker-compose-run